interpreter/text/arrange.ml

```diff
@@ -205,11 +205,14 @@ let rmw = function
   | I32 I32Op.RmwXchg | I64 I64Op.RmwXchg -> "xchg"
   | _ -> assert false
 
-let memop name {ty; align; offset; _} =
-  value_type ty ^ "." ^ name ^
+let memarg {ty; align; offset; _} =
```

Comment:
Nit: since this one outputs a proper memop, I'd rather keep its name, and call the new one something different (`memop_without_type`?). Also, I think you can keep the name argument, since even the new function prepends it.

Reply:
Done